test/Microsoft.AspNetCore.Hosting.Tests/Fakes/Startup.cs

```diff
@@ -7,16 +7,17 @@
 
 namespace Microsoft.AspNetCore.Hosting.Fakes
 {
-    public class Startup : StartupBase
+    public class Startup : StartupBase, IStartup
     {
         public Startup()
         {
         }
 
-        public void ConfigureServices(IServiceCollection services)
+        public IServiceProvider ConfigureServices(IServiceCollection services)
```

Comment:
So that I can use it in the test at https://github.com/aspnet/Hosting/blob/57634a8dd91852a9f62d49f078260b604ed389ec/test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs#L589